docs/concepts/what-is-an-activity.md

```diff
@@ -11,7 +11,7 @@ tags:
 In day-to-day conversations, the term _Activity_ frequently denotes either an [Activity Definition](/concepts/what-is-an-activity-definition), an [Activity Type](/concepts/what-is-an-activity-type), or an [Activity Execution](/concepts/what-is-an-activity-execution).
 Temporal documentation aims to be explicit and differentiate between them.
 
-An Activity is a normal function or object method that executes a single, well-defined action (either short or long running), such as calling another service, transcoding a media file, or sending an email message.
+Temporal recommends that Activities are idempotent. Activities are idempotent if multiple applications of that operation do not change the state of the system beyond the initial application.
```

Comment:
```suggestion
Temporal recommends that Activities be idempotent. Activities are idempotent if multiple [Activity Task Executions](/concepts/what-is-an-activity-task-execution) do not change the state of the system beyond the first Activity Task Execution.
```

Reply:
actually, since we don't have independent glossary pages anymore, I wonder whether we should remove this instance, since the same words will appear lower down on the activities page. https://docs.temporal.io/activities

also, I liked the deleted sentence:

An Activity is a normal function or object method that executes a single, well-defined action (either short or long running), such as calling another service, transcoding a media file, or sending an email message.

it gave me a practical idea of what an Activity is, which is harder to get if I read the page without it